Make-a-pull-request.md

```diff
@@ -90,7 +90,9 @@ Once your feature is ready, you can open a pull request (PR)!
 
 * Go to your fork on GitHub, select your branch from the dropdown menu, and click "pull request". Ensure that the base repository is `oppia/oppia` and that the base branch is `develop`. The head repository should be your fork, and the head branch should be your branch. If you don't see the repository, click the link to `compare across forks`.
 
-  On this page you can also see your commits and your changes.  Read these changes carefully to make sure that the changes are correct (e.g., no missing newlines at the ends of files; no files left out by mistake). This is a good way to catch obvious errors that would otherwise lead to delays in the review process.
+  On this page, you can also see your commits and your changes.
+
+  **Important:** **Check those changes carefully** to make sure that they are correct (e.g., no extra files; no files left out by mistake; no missing newlines at the ends of files). This is a good way to catch obvious errors that would otherwise lead to delays in the review process. (You can also double-check them on the "Files Changed" tab after the PR is created, but make sure to do at least one self-review before assigning any reviewers.)
```

Comment:
```suggestion
> [!WARNING]
> **Important:** **Check those changes carefully** to make sure that they are correct (e.g., no extra files; no files left out by mistake; no missing newlines at the ends of files). This is a good way to catch obvious errors that would otherwise lead to delays in the review process. (You can also double-check them on the "Files Changed" tab after the PR is created, but make sure to do at least one self-review before assigning any reviewers.)
```

I think using GitHub Markdown's [alerts syntax](https://github.com/orgs/community/discussions/16925) will make this warning harder to miss. In my testing, warnings could not be indented, so you might need to move this outside the list of bullet points

Reply:
Done, thanks @U8NWXD. PTAL.